### ISSUE TITLE
Netbeans 8 support

### ISSUE THREAD

**Issue by chippyash** [QuBiT/cucumber-netbeans-plugin#33]
The 1.8.2. module for Netbeans 7.1 installs into Netbeans 8.0 OK, but when you restart Netbeans, the following error occurs during the startup process

Warning - could not install some modules: Cucumber Features - The module named org.netbeans.modules.languages/0-1 was needed and not found.

**Comment by lc-nyovchev** [comment]
I don't get the same problem, but running Netbeans 8.1 and maybe have the needed language module.

**Comment by Eidansoft** [comment]
For me, version 1.8.2 downloaded from http://plugins.netbeans.org/plugin/43852/cucumber-features seems to work at Netbeans 8.0.2 (English). The only thing I'm not happy is to open the implementation method with Ctrl+click on the Gherkin step at a feature file. But I'm not sure if that behavior exists at the  plugin on other Netbeans versions.

**Comment by mickael28** [comment]
>> The only thing I'm not happy is to open the implementation method with Ctrl+click on the Gherkin step at a feature file. But I'm not sure if that behavior exists at the plugin on other Netbeans versions.

* Eidansoft, do you mean that if you click CTRL+click on the feature file it goes to the implementation of that step? I don't manage to navigate to the step definition in any way... I've clicked all combinations of mouse buttons and keyboard keys, plus tried a few different Netbeans versions and still not successful.

When I click, nothing happens... so I cannot navigate anywhere from the feature files

**Comment by Eidansoft** [comment]
No, I mean that Ctrl+click do NOT work. Sorry for the misunderstanding.
